### ISSUE TITLE
Unable to get email to send

### ISSUE THREAD

**Issue by blackbyrd84** [adamcooke/staytus#318]
Hello,

I am unable to get email to send unless I am in the ruby console and calling send_verification_email, or you subscribe from the main page. Any time a new Issue is posted/updated or Maintenance is posted/updated, emails are never sent regardless if the subscriber was entered manually from admin or subscribed from the front end. I can see in the log that the job is played in the delayed jobs table, but the table is always empty. I followed [#256 ](https://github.com/adamcooke/staytus/issues/256) which led me to using systemd to start procodile on boot, which still did not resolve the issue. Not certain how the verification emails are able to be sent, but no other. I am using nginx per your guide to use 80/443 with a wildcard cert, if that makes any difference.

**Comment by Justin417** [comment]
We're having the same exact problem. I can force verification emails without issue, but issue notifications simply fall off the face of the earth. 

Our setup has been running for over a year without issues, we just now has this issue occur randomly without any changes to the system.

Has anyone found a workaround?

**Comment by SuperSajuuk** [comment]
I'm also having this issue. I use Phusion Passenger to run the local webserver (due to its integration with cPanel/WHM/Apache meaning that it will handle booting as needed), did anyone find a way to fix this after configuring the environment variables as required?

**Comment by Justin417** [comment]
Unfortunately we haven't found a solution yet (at the moment we have restored to bulk-emailing manually, not fun). 

However we have noticed that initial new issue messages are being sent successfully, only issue update emails aren't sent. 

I've spent quite some time digging around our system but haven't noticed any changes or issues that would have provoked this issue. I'm no Ruby developer by any means so my troubleshooting abilities have been limited.
